kdave/xfstests
# generic/260: fix unary operator error when run tests

When I run test/generic/260 with check script at ext4 filesystem, there is error that appeared at the terminal like below. So I try to add bracket and run again, the error disappeared.

QA output created by 260
[+] Start beyond the end of fs (should fail)
fstrim: SCRATCH_MNT: FITRIM ioctl failed: Invalid argument
[+] Start beyond the end of fs with len set (should fail)
fstrim: SCRATCH_MNT: FITRIM ioctl failed: Invalid argument
[+] Start = 2^64-1 (should fail)
fstrim: SCRATCH_MNT: FITRIM ioctl failed: Invalid argument
[+] Start = 2^64-1 and len is set (should fail)
fstrim: SCRATCH_MNT: FITRIM ioctl failed: Invalid argument
[+] Default length (should succeed)
[+] Default length with start set (should succeed)
[+] Length beyond the end of fs (should succeed)
[+] Length beyond the end of fs with start set (should succeed)
/var/lib/xfstests/tests/generic/260: line 89: [: -gt: unary operator expected
/var/lib/xfstests/tests/generic/260: line 102: [: -le: unary operator expected
/var/lib/xfstests/tests/generic/260: line 174: [: -le: unary operator expected
Test done
